tests/index_test.js

```diff
@@ -15,11 +20,14 @@ after(() => { console.error.restore() })
 
 // setup jsdom
 beforeEach(() => {
+  const fetch = window.fetch
   jsdom.env({
     url: 'https://data.example.com/',
     html: '<div></div>',
     created: (err, window) => {
       if (!err) {
+        window.fetch = fetch
+        global.fetch = fetch
```

Comment:
This is pretty gross; I'm not sure what the right way to fix this is, but upgrading somehow made it so we need both of these lines.

Reply:
Agreed